packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -457,3 +551,37 @@ test('copyWrappedGlobals - copy from prototype too', (t) => {
 
   t.is(Object.keys(target).sort().join(), 'onTheObj,onTheProto,window')
 })
+
+test('copyWrappedGlobals+endowAll - ends up including fields from prototype too', (t) => {
+  'use strict'
+  const { copyWrappedGlobals, endowAll } = prepareTest()
+  const sourceProto = {
+    onTheProto: function () {},
+  }
+  const source = Object.create(sourceProto)
+  source.onTheObj = function () {}
+  const target = Object.create(null)
+  copyWrappedGlobals(source, target, ['window'])
+  const endowments = endowAll(target, {}, {})
+
+  t.is(Object.keys(endowments).sort().join(), 'onTheObj,onTheProto,window')
```

Comment:
[suggestion] arguably reduces cognitive overhead and maintenance burden:

```suggestion
  t.like(Object.keys(endowments).sort(), ['onTheObj', 'onTheProto', 'window'])
```

(or `t.deepEqual()`)

Reply:
In my experience the test output of comparing arrays is horrible. I was considering switching to a string comparison in webpack dist file snapshots because the comparison is so unreaedble when they're an array.
In this case it doesn't really matter for 3 elements.

I don't want to use `like` because it leaves room for flexibility. Might be convinced to use `deepEqual`.

Reply:
fair enough. I want to switch to `node:test` at some point anyhow